src/styles/styles.js

```diff
@@ -2028,12 +2036,24 @@ const styles = {
     avatarInnerTextChat: {
         color: themeColors.textLight,
         fontSize: variables.fontSizeNormal,
-        left: 1,
+        left: 2,
         textAlign: 'center',
         fontWeight: 'normal',
         position: 'absolute',
     },
 
+    SVGAvatarBorder: {
```

Comment:
All styles are camelCase, but this one is TitleCase

Reply:
I think this is technically okay because of the [acronym pattern](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms). However, looking at `svgAvatarBorder` vs `SVGAvatarBorder`, I think I prefer all lowercase for readability